### PR TITLE
Remove parent attribute from xdita

### PR DESCRIPTION
**Pull request by marmoure**
Prevent the generation process from adding the parent attribute in `xdita`

### CODE REVIEW

File: packages/prosemirror-lwdita-demo/src/demo-plugin.ts [hunk 1]
```diff
@@ -140,7 +140,7 @@ function saveFile(input: InputContainer): Command {
   return (state: {[x: string]: any; tr: any; selection: { empty: any; };}, dispatch: (arg0: any) => void) => {
     if (dispatch) {
       dispatch(state.tr);
-      const xditaPrefix = `<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE topic PUBLIC "-//OASIS//DTD LIGHTWEIGHT DITA Topic//EN" "lw-topic.dtd">`;
+      const xditaPrefix = `<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE topic PUBLIC "-//OASIS//DTD LIGHTWEIGHT DITA Topic//EN" "lw-topic.dtd">\n`;
```
Comment (adamretter) [inline packages/prosemirror-lwdita-demo/src/demo-plugin.ts:143]: @marmoure there needs to be a `\n` after the xml declaration please.

File: packages/prosemirror-lwdita/src/document.ts [hunk 2]
```diff
@@ -301,6 +301,7 @@ export function unTravel(prosemirrorDocument: Record<string, any>): JDita {
   }
 
   // handle the attributes
+  delete attributes['parent'];
```
Comment (adamretter) [inline packages/prosemirror-lwdita/src/document.ts:304]: What does this do? It doesn't seem to be tested for anywhere?

**Review by adamretter — CHANGES_REQUESTED** [review]

File: packages/prosemirror-lwdita/src/document.ts (hunk 2 above)
Comment (marmoure) [reply packages/prosemirror-lwdita/src/document.ts:304]: > It doesn't seem to be tested for anywhere?

The cypress test should cover this one!!

File: packages/prosemirror-lwdita-demo/src/demo-plugin.ts (hunk 1 above)
Comment (marmoure) [reply packages/prosemirror-lwdita-demo/src/demo-plugin.ts]: Done!

File: packages/prosemirror-lwdita/src/document.ts (hunk 2 above)
Comment (adamretter) [reply packages/prosemirror-lwdita/src/document.ts:304]: @marmoure The test is passing before this change! Therefore there is no test for this...

Comment (marmoure) [reply packages/prosemirror-lwdita/src/document.ts:304]: https://github.com/evolvedbinary/prosemirror-lwdita/commit/aedb23f171d776fdd93b20c28352055353ee7ce8
there was a test, I not sure how this got approved!!

Comment (adamretter) [reply packages/prosemirror-lwdita/src/document.ts:304]: @marmoure that may have been my fault - I didn't mean to add the `.skip` can you remove the skip and check it works please

**Review by adamretter — APPROVED** [review]

**Pull request merged**